src/Symfony/Component/Messenger/Command/FailedMessagesShowCommand.php

```diff
@@ -36,6 +36,7 @@ protected function configure(): void
             ->setDefinition([
                 new InputArgument('id', InputArgument::OPTIONAL, 'Specific message id to show'),
                 new InputOption('max', null, InputOption::VALUE_REQUIRED, 'Maximum number of messages to list', 50),
+                new InputOption('failed-transport', null, InputOption::VALUE_OPTIONAL, 'Use a specific failed transport'),
```

Comment:
All of these should probably be `failure-transport`, right? That's what we call it in the config